### PR TITLE
YONK-16: Added spilt module store for tests

### PR DESCRIPTION
**Pull request by aamishbaloch**

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: course_metadata/tests/tests.py [hunk 1]
```diff
@@ -23,7 +29,7 @@ def setUp(self):
         self.chapter = ItemFactory.create(
             category="chapter",
             parent_location=self.course.location,
-            data=self.test_data,
+            # data=self.test_data,
```
Comment (ziafazal) [inline course_metadata/tests/tests.py]: commented code.

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch could you please remove commented code.

**Comment by aamishbaloch** [comment]
@ziafazal removed the commented code. Can you review it?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**